internal/api/router.go

```diff
@@ -162,6 +162,7 @@ func NewRouter(handlers Handlers, cfg *config.Configuration, logger *logger.Logg
 			invoices.POST("/:id/void", handlers.Invoice.VoidInvoice)
 			invoices.PUT("/:id/payment", handlers.Invoice.UpdatePaymentStatus)
 			invoices.POST("/preview", handlers.Invoice.GetPreviewInvoice)
+			invoices.POST("/generate", handlers.Invoice.GenerateInvoice)
```

Comment:
Is this GenerateInvoice same as the `handlers.Cron.GenerateInvoice`? If yes then lets just keep the cron endpoint one for now